health.go

```diff
@@ -293,6 +294,12 @@ func GetSysErrors(ctx context.Context, addr string) SysErrors {
 		se.Errors = append(se.Errors, SysErrUpdatedbInstalled)
 	}
 
+	o, err := exec.Command("getenforce").Output()
```

Comment:
There are many ways to check `SELinux`. My recommendation is to use `/etc/selinux/config` file.
1. If `/etc/selinux/config` is not found, `SELinux` is not installed.
2. If `/etc/selinux/config` contains a line `SELINUX=enforcing`, `SELinux` is enabled.
3. If `/etc/selinux/config` contains a line `SELINUX=permissive`, `SELinux` is running in warning mode i.e. it doesn't fail, but logs illegal access.
4. If `/etc/selinux/config` contains a line, `SELINUX=disabled`, `SELinux` is disabled.

Here, we shouldn't fail for (3), instead of raise a warning.

Reply:
@harshavardhana are two levels (warning and error) really required here? The current implementation records anything that is not recommended as an error. Any entry in `SysErrors` will show up in red on the subnet health UI.